src/apps/lwaftr/alarms/lwaftr-default-alarms.conf

```diff
@@ -0,0 +1,18 @@
+alarms {
+   alarm-inventory {
+       alarm-type {
+           alarm-type-id arp-resolution;
+           alarm-type-qualifier "";
+           has-clear true;
+           resource external-interface;
```

Comment:
alarm-qualifier is optional, so I left it blank. According to ietf-alarms, alarm-qualifier is defined as "The optionally dynamically defined alarm type identifier for this possible alarm.". So far I didn't need to use it.

I think I should state something in the description.

I agree with your point of view. Ideally the alarms should be defined by an app. Because actually in the case of ARP, there's code in the module to raise a "arp-resolution" alarm, so it makes sense that the ARP app also contains defines its alarm-types in a yang file.